HorizonEngine/src/HorizonEngine/Logging/Logging.h

```diff
@@ -24,6 +27,21 @@ namespace Hzn
 
 }
 
+template<>
+struct fmt::formatter<Hzn::Event> {
+	constexpr auto parse(format_parse_context& ctx) -> decltype(ctx.begin()) {
+		return ctx.end();
+	}
+	
+
+	template <typename FormatContext>
```

Comment:
actually:
just writing
```
template<>
struct fmt::formatter<Hzn::Event> : fmt::ostream_formatter {};
```
this much does the job.

Reply:
Thank you for your suggestion, this is a better solution.